opencensus/exporters/trace/stackdriver/stackdriver_exporter.h

```diff
@@ -31,12 +33,18 @@ struct StackdriverOptions {
 
   // The RPC deadline to use when exporting to Stackdriver.
   absl::Duration rpc_deadline = absl::Seconds(5);
+
+  // (optional) By default, the exporter connects to Stackdriver using gRPC. If
+  // this stub is non-null, the exporter will use this stub to send gRPC calls
+  // instead. Useful for testing.
+  std::unique_ptr<google::devtools::cloudtrace::v2::TraceService::Stub>
+      trace_service_stub;
 };
 
 class StackdriverExporter {
  public:
   // Registers the exporter.
-  static void Register(const StackdriverOptions& opts);
+  static void Register(StackdriverOptions&& opts);
```

Comment:
Is this not a breaking change? What about just accept pass by value?

Reply:
Yeah it's a breaking change - see the diffs in dir examples/grpc/...

We can't pass StackdriverOptions by value if there's a unique_ptr in there.

We could pass it by ref and swap out the contents. (option 1)

Or put a plain pointer in StackdriverOptions and document it as transferring ownership. (option 2)

What do you think?